handel/src/services/iot/index.ts

```diff
@@ -86,21 +86,21 @@ async function deleteTopicRule(ruleName: string) {
  */
 
 export function check(serviceContext: ServiceContext<IotServiceConfig>, dependenciesServiceContexts: Array<ServiceContext<ServiceConfig>>) {
-    const errors = [];
-
+    const errors: string[] = checkPhase.checkJsonSchema(`${__dirname}/params-schema.json`, serviceContext);
+    
     const serviceParams = serviceContext.params;
     if (serviceParams.event_consumers) {
```

Comment:
I wasn't sure if these if statements needed to be removed, so I left them in

Reply:
Oh yes I need to talk to you about these event_consumers. They are one of two special parameter types that can be applied to multiple services, so in this case we'll need to include them in the JSON schema.